src/HDInsight/HDInsight/ManagementCommands/SetAzureHDInsightGatewayCredentialCommand.cs

```diff
@@ -0,0 +1,95 @@
+﻿// ----------------------------------------------------------------------------------
+//
+// Copyright Microsoft Corporation
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ----------------------------------------------------------------------------------
+
+using Microsoft.Azure.Commands.HDInsight.Commands;
+using Microsoft.Azure.Commands.ResourceManager.Common.ArgumentCompleters;
+using Microsoft.Azure.Management.HDInsight.Models;
+using Microsoft.WindowsAzure.Commands.Common;
+using System;
+using System.Management.Automation;
+
+namespace Microsoft.Azure.Commands.HDInsight
+{
+    [Cmdlet(VerbsCommon.Set, ResourceManager.Common.AzureRMConstants.AzureRMPrefix + "HDInsightGatewayCredential", DefaultParameterSetName = SetGatewayCredentialParameterSetName, SupportsShouldProcess = true), OutputType(typeof(HttpConnectivitySettings))]
```

Comment:
Please see the cmdlet review for notes

Reply:
> Please see the cmdlet review for notes

Have  updated the cmdlet based by design guides.

Reply:
> Please see the cmdlet review for notes

Have update cmdlet issue

Reply:
fixed

---
In reply to: [278679287](https://github.com/Azure/azure-powershell/pull/9069#discussion_r278679287) [](ancestors = 278679287)